modules/doc/content/getting_started/installation/conda.md

```diff
@@ -7,7 +7,7 @@ an option to install MOOSE directly on a Windows system is not yet supported.
 On-going efforts are being made to add a conda installation option for Windows,
 and an experimental [WSL](installation/windows10.md) option is available.
 
-!include installation/install_miniconda.md
+!include installation/install_mambaforge.md
```

Comment:
Because of the current error for "[Moose Install Target Test](https://civet.inl.gov/job/900508/)" I am wondering if we should rename this to simply 'install_conda.md'.

[github.com mastodon mastodon_conda.md](https://github.com/idaholab/mastodon/blob/650ffff487934ed6215cb2dc11124910a3873dee/doc/content/getting_started/mastodon_conda.md?plain=1#L17)

So we don't run into this again when/if we happen to switch to some other conda installer. There might be other application documentations which is referencing this file name.